post/models.py

```diff
@@ -25,7 +25,8 @@ class Comment(models.Model):
     post = models.ForeignKey(Post, related_name="comments", on_delete=models.CASCADE)
     name = models.CharField(max_length=100)
```

Comment:
Can you please change the name here to same as Marie is using in Post model and make it a ForeignKey same way as she did so it will be consistent?

Reply:
changed